pom.xml

```diff
@@ -293,12 +308,16 @@
             </resource>
         </resources>
         <plugins>
-
+            <plugin>
```

Comment:
Commenting here just to make sure that you intended to leave this in. I thought that we were only adding this while upgrading the versions, but then we were removing it. It's not a big deal either way.

Reply:
removed the versions plugin